File: dart-jaguar/lib/model/types_ipfs_pin.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/object.dart';
part 'types_ipfs_pin.jser.dart';

class TypesIpfsPin {
  
  final String cid;
  
  final Object meta;
  
  final String name;
  
  final List<String> origins;
  

  TypesIpfsPin(
    

{
     this.cid = null,  
     this.meta = null,  
     this.name = null,  
     this.origins = const [] 
    
    }
  );

  @override
  String toString() {
    return 'TypesIpfsPin[cid=$cid, meta=$meta, name=$name, origins=$origins, ]';
  }
}

@GenSerializer()
class TypesIpfsPinSerializer extends Serializer<TypesIpfsPin> with _$TypesIpfsPinSerializer {

}

File: scala-akka-http-server/src/main/scala/io/swagger/server/model/Types.IpfsPin.scala
package io.swagger.server.model


/**
 * @param cid 
 * @param meta 
 * @param name 
 * @param origins 
 */
case class Types.IpfsPin (
  cid: Option[String],
  meta: Option[Object],
  name: Option[String],
  origins: Option[List[String]]
)

